packages/field/src/react/label.tsx

```diff
@@ -18,7 +18,9 @@ const styles = {
     )
 }
 
-interface LabelProps extends HTMLPropsFor<'label'> {}
+interface LabelProps extends HTMLPropsFor<HTMLLabelElement> {
+  htmlFor?: string
```

Comment:
I throws an error that's why. I don't know why since it actually exist on the HTMLLabelElement but it could have been vscode throwing a false flag. Try removing it in your env and running a build for field or I will tomorrow.